.github/workflows/scanner-release-vuln-update.yaml

```diff
@@ -3,22 +3,29 @@ on:
   schedule:
   - cron: "0 */3 * * *"
   workflow_dispatch:
+    inputs:
+      upload:
+        description: Upload generated vulnerabilities to the production bucket.
+        required: false
+        type: boolean
 
 jobs:
   read-release-versions:
     runs-on: ubuntu-latest
     outputs:
       matrix: ${{ steps.output-matrix.outputs.matrix }}
     steps:
-    - name: Checkout code
+    - name: Checkout
       uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
 
     - name: Generate matrix JSON
       id: output-matrix
       run: |
         EOF=$(dd if=/dev/urandom bs=15 count=1 status=none | base64)
         echo "matrix<<$EOF" >> "$GITHUB_OUTPUT"
-        sed -n '/^\s*\(#.*\)\?$/!p' scanner/updater/version/RELEASE_VERSION | jq -Rs '{ versions: ( sub("\n$"; "")|split("\n") ) }' | tee -a "$GITHUB_OUTPUT"
+        ./.github/workflows/scripts/scanner-output-release-versions.sh | tee -a "$GITHUB_OUTPUT"
```

Comment:
do we need to do this for offline as well?

Reply:
Can you clarify why you think we should? I don't see why we should do this for the offline bundle.

Reply:
I was just wondering